pkg/skaffold/initializer/init.go

```diff
@@ -54,6 +54,18 @@ type Initializer interface {
 	GetImages() []string
 }
 
+// InitBuilder represents a builder that can be chosen by skaffold init.
+type InitBuilder interface {
```

Comment:
_Effective Go_ seems to recommend [avoiding prefixing getters with _Get_](https://golang.org/doc/effective_go.html#Getters).  I'll put some suggestions below.